src/lucky/text_response.cr

```diff
@@ -20,6 +20,8 @@ class Lucky::TextResponse < Lucky::Response
   end
 
   def print : Nil
+    write_session
+    write_cookies
```

Comment:
Now that cookies/session are set here, I think the flash handler should also be removed and the flash set before `write_session`. That way it gets written to the session cookie when it is printed.

Reply:
Great, I'll do that as well.

Reply:
Voila, that's done!